meson.build

```diff
@@ -0,0 +1,21 @@
+project('mididings', 'cpp', default_options: ['cpp_std=c++17'])
```

Comment:
I would suggest adding:

```meson
meson_version: '>=0.64.0'
```

since you require at least that version anyway, and users of stable distros might have an older version by default.